biolink-model.yaml

```diff
@@ -202,7 +202,105 @@ enums:
         description: nucleic acid
       "AA":
         description: amino acid
-        
+
+  chemical_available_from_enum:
```

Comment:
maybe "drug availability" is more precise then "chemical available from"

Reply:
done! :) thanks a bunch.